app/indexers/default_work_indexer.rb

```diff
@@ -17,13 +17,19 @@ def generate_solr_document
       license_labels = ScholarsArchive::LicenseService.new.all_labels(object.license)
       language_labels = ScholarsArchive::LanguageService.new.all_labels(object.language)
       peerreviewed_label = ScholarsArchive::PeerreviewedService.new.all_labels(object.peerreviewed)
+      academic_affiliation_labels = ScholarsArchive::TriplePoweredService.new.fetch_top_label(object.academic_affiliation, parse_date:true)
```

Comment:
Both of these calls are covered already by the `object.triple_powered_properties...` block below. 

Instead of these two calls, maybe update the block below to be like;
 ```
object.triple_powered_properties.each do |o|
  labels = ScholarsArchive::TriplePoweredService.new.fetch_top_label(object.send(o[:field]), o[:has_dates])
  solr_doc[o[:field].to_s + '_label_ssim'] = labels
  solr_doc[o[:field].to_s + '_label_tesim'] = labels
end